### PR TITLE
1.5.0 release note

### PR DESCRIPTION
**Pull request by jholdstock**

### CODE REVIEW

File: docs/release-note-1.5.0.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+1. Build dcrwallet and dcrd v1.5.0
+1. Announce maintenance and shut down dcrstakepool
+1. Perform an upgrade of each back-end server, one at a time
+   1. Stop stakepoold
+   1. Stop dcrwallet
+   1. Stop dcrd
+   1. Install latest dcrd binary and start
+   1. Install latest dcrwallet binary and start
+   1. Make required changes to stakepoold.conf (detailed [above](#config-changes))
+   1. Install latest stakepoold binary and start
```
Comment (JoeGruffins) [inline docs/release-note-1.5.0.md]: I feel like we should say something about not starting stakepoold until dcrd and dcrwallet have synced with the network.  I think we had a problem here last time everyone updated. Or is there anything we could do in the code? should make an issue?

**Comment by jholdstock** [comment]
Thanks Joe. Implemented both suggestions.

I actually thought stakepoold could handle the scenario where dcrd/dcrwallet are not fully up to date. This may be worth investigating further, because it really should be able to handle it.

**Comment by JoeGruffins** [comment]
I think the scenario was thus, the operator updated their vsp, they started everything at once. Right after the pool coming online, a new user links their account to a wallet and buys some tickets, the wallet is currently syncing, and the redeem script is not imported until the initial sync is over. However, the bought tickets are in that initial sync, so the wallet passes them over. I will try to reproduce this, I'll make an issue if I can.  The conversation about this is in the pos room on Tue, Nov 26 2019

**Comment by JoeGruffins** [comment]
So far unable to reproduce... iirc the solution was to rescan their wallets.  So they had the redeem script, but for some reason the wallets, all of them, had missed the ticket purchase.  It was like 10 tickets purchased by the same new user.

**Comment by jholdstock** [comment]
Well it's included in the steps now anyway. Better to be safe than sorry. Hopefully it prevents somebody else running into issues.

**Pull request merged**